packages/truffle/index.js

```diff
@@ -0,0 +1 @@
+console.log(__dirname);
```

Comment:
Why's this necessary?

Reply:
oof, honestly I don't know where this came from. Good catch!